grails-app/views/purchaseOrder/_showOrderItems.gsp

```diff
@@ -471,7 +471,7 @@
             var data = $("#orderItemForm").serialize();
             data += '&orderIndex=' + $("#orderItemsTable tbody tr").length;
             if (validateItemsForm()) {
-              if ($("#validationCode").val() == 'WARN' && !confirm('${warehouse.message(code: 'orderItem.warningSupplier.label')}')) {
+              if ($("#validationCode").val() == 'WARN' && !confirm('${warehouse.message(code: 'orderItem.warningSupplier.label').replaceAll( /(['])/, '\\\\$1' )}')) {
```

Comment:
I think, that `.replaceAll( /(')/, '\\\\$1' )` will suffice, no need for `[ ]`

Reply:
@awalkowiak you're right, done.